docs/v3.x/concepts/configurations.md

```diff
@@ -50,6 +50,26 @@ module.exports = ({ env }) => {
 
 ## Environment variables
 
+### List of Strapi's environment variables
+
+Some settings can only be modified through environment variables. Here is a list of those settings are associated environment variable names:
+
+| name                                 | description                                                                        | type    | default                   |
+| ------------------------------------ | ---------------------------------------------------------------------------------- | ------- | ------------------------- |
+| `NODE_ENV`                           | Type of environment where the app is running                                       | string  | `'development'`           |
+| `STRAPI_DISABLE_UPDATE_NOTIFICATION` | Don't show the notification message about updating strapi in the terminal          | boolean | `false`                   |
+| `STRAPI_HIDE_STARTUP_MESSAGE`        | Don't show the startup message in the terminal                                     | boolean | `false`                   |
+| `STRAPI_TELEMETRY_DISABLED`          | Don't send telemetry usage data to Strapi                                          | boolean | `false`                   |
+| `BROWSER`                            | Open the admin panel in the browser after startup                                  | boolean | `true`                    |
+| `STRAPI_LICENSE`                     | The license key to activate the Enterprise Edition                                 | string  | `undefined`               |
+| `ENV_PATH`                           | Path to the file that contains your environment variables                          | string  | `'./.env'`                |
+| `STRAPI_LOG_TIMESTAMP`               | Add the timestamp info in logs                                                     | boolean | `false`                   |
+| `STRAPI_LOG_LEVEL`                   | Select the level of logs among `fatal`, `error`, `warn`, `info`, `debug`, `trace`. | string  | `'info'`                  |
+| `HOST`                               | Host on which the app runs                                                         | string  | `'0.0.0.0' | 'localhost'` |
+| `PORT`                               | Port on which the app runs                                                         | number  | `1337`                    |
+
```

Comment:
@alexandrebodin @derrickmehaffy Can you check that all those line are legit?
Maybe we don't want to add in the documentation some of those variables (in case we want to rename them for example)

Reply:
a few comments:
- I would group the ones prefixed with STRAPI. 
- I think the startup message an log envs are already documented so we might need to remove them somewhere else.
- I would remove HOST and PORT as they are only set in the config files so they can be removed at any point by the user